content/docs/concepts/architecture.md

```diff
@@ -0,0 +1,320 @@
++++
+title = "Architecture"
+weight = 3
+date = "2021-06-21"
++++
+
+## Key Concepts
+### Network Services
+
+A Network Service is a set of Connectivity, Security, and Observability features at L3 and above to which workloads may individually connect.
+
+![Network Service Venn Diagram](/img/concepts/architecture/ns_venn.svg)
+
+A Network Service is a collection of Connectivity, Security, and Observability features applied to traffic.
+
+Examples of Network Services would include:
+- A simple distributed vL3 that allows the workloads to communicate via IP, optionally with DNS service for that vL3
+- A Traditional Network Service like Istio, Linkerd, Consul, or Kuma running over a vL3.  This allows specific workloads to be admitted to that Service Mesh, independent of where they run.
+It also allows a single workload to connect to multiple Traditional Service Meshes.  This can allow a workload to connect both to a companies Service Mesh, and also to the Service Meshes of its partners simultaneously.
```

Comment:
"This can allow a workload to connect to both a company's service mesh and that of a partner's simultaneously."